doc/build-android.md

```diff
@@ -22,4 +22,233 @@ This is an example command for a default build with no disabled dependencies:
+2. Click the `SDK Tools` tab.
+3. Select the `NDK (Side by side)` and `CMake` checkboxes.
+4. Click `OK`.
+5. A dialog box will tell you how much space the NDK package consumes on disk.
+6. Click `OK`.
+7. When the installation is complete, click `Finish`.
+
+You can find the full guide [here](https://developer.android.com/studio/projects/install-ndk).
+
+### Step 5: Check Android Device Architecture
+
+Before you proceed, ensure you check your Android Device's hardware architecture. Use usbip and adb for this.
+
+1. Connect your Android device to your PC and enable USB debugging in Developer Options.
+2. Once usbip is installed, list all USB devices connected to your PC by running the following command as Administrator in `cmd`:
+
+    ```bash
+    usbipd list
+    ```
+
+    If it doesn't say `SHARED` under the `STATE` collumn then run:
+    ```bash
+    usbipd bind -b <busid>
+    ```
+
+3. Note down the `BUSID` of the device you want to connect to WSL. Then run the following command, replacing `<busid>` with the `BUSID` of your device:
+
+    ```bash
+    usbipd attach --busid <busid> --wsl
+    ```
+
+4. Install adb:
+
+    ```bash
+    sudo apt install adb
+    ```
+
+5. Check the hardware architecture of your device:
+
+    ```bash
+    adb shell getprop ro.product.cpu.abi
+    ```
+
+6. Note down the architecture (arm64-v8a, armeabi-v7a, x86_64, x86).
+
+### Step 6: Install Gradle
+
+1. Download Gradle 6.6.1:
+
+    ```bash
+    VERSION=6.6.1
+    wget https://services.gradle.org/distributions/gradle-${VERSION}-bin.zip -P /tmp
+    ```
+
+2. Extract the file:
+
+    ```bash
+    sudo unzip -d /opt/gradle /tmp/gradle-${VERSION}-bin.zip
+    ```
+
+3. Set environment variables:
+
+    ```bash
+    sudo nano /etc/profile.d/gradle.sh
+    ```
+
+4. Add the following lines to the file:
+
+    ```bash
+    export GRADLE_HOME=/opt/gradle/gradle-${VERSION}
+    export PATH=${GRADLE_HOME}/bin:${PATH}
+    ```
+
+5. Change the permissions:
+
+    ```bash
+    sudo chmod +x /etc/profile.d/gradle.sh
+    ```
+
+6. Load the environment variables:
+
+    ```bash
+    source /etc/profile.d/gradle.sh
+    ```
+
+7. Verify the installation by checking the Gradle version:
+
+    ```bash
+    gradle -v
+    ```
+
+You can follow the full guide to install Gradle [here](https://linuxhint.com/installing_gradle_ubuntu/).
+
+### Step 7: Build APKs
+
+Before building the APKs, run the following commands:
+
+```bash
+PATH=$(echo "$PATH" | sed -e 's/:\/mnt.*//g') # strip out problematic Windows %PATH% imported var
+sudo bash -c "echo 0 > /proc/sys/fs/binfmt_misc/status" # Disable WSL support for Win32 applications.
+```
+
+More details on this step can be found [here](https://github.com/bitcoin/bitcoin/blob/master/doc/build-windows.md#compiling-with-windows-subsystem-for-linux).
+
+Now, you can build the APKs using the guide found [here](https://github.com/bitcoin-core/gui-qml/blob/main/doc/build-android.md).
```

Comment:
nit:

```suggestion
Now, you need to build the dependencies for the specific Android API Level, for which you can follow the guide found [here](https://github.com/bitcoin-core/gui-qml/blob/main/doc/build-android.md).
```

Maybe that guide above needs to be updated as it shows an example for "a default build with no disabled dependencies", and could display the `make -C depends` suggested previously. And later below you also specify run `make && make apk` from `src/qt` as on the build-android guide.

In my opinion, I'd be more clear on the sub-steps here in Step 7 in order to build the APK and transfer/ install it to the device (including for each the other commands to make it work as the `qlogging.cpp` update, paths, etc):
1. build dependencies
2. configure with proper host
3. make
4. make apk
5. adb install

Reply:
Great idea on the having sub-steps